components/logins/src/lib.rs

```diff
@@ -10,20 +10,21 @@ mod error;
 mod login;
 
 mod db;
-pub mod schema;
+mod schema;
 mod store;
 mod update_plan;
 mod util;
 
-mod ffi;
+// Following how fxa-crate hides some of the implementation details we don't want to expose
```

Comment:
(Although I think that's largely defeated by the `pub use crate::error::*` here and the `pub use crate::internal::error::*;` in `error.rs`)

Reply:
I agree it adds much more complexity for no real gain. Will move it back in to the top-level!

Reply:
FWIW pushing it back up to the top-level sounds fine to me, I think we're still learning about the best patterns to use for these things.